### DISCUSSION TITLE
Configuring Namespace in values.yaml for Strimzi Cluster Operator Deployment

### DISCUSSION THREAD

**Discussion by nikhilmulinti** [strimzi/strimzi-kafka-operator#9246] [General]
I've successfully deployed the strimzi-cluster-operator using the following command:
helm install strimzi-cluster-operator ./strimzi-kafka-operator -n cluster-kafka
Is it possible to specify the namespace directly within the values.yaml file? I'm aware that hardcoding the namespace isn't recommended. However, I'm using a Golang script to manage this deployment, and it doesn't allow me to provide the -n parameter during deployment.

**Comment by naresh-egov** [comment]
I am also facing the same issue.
